hooks/pre-push.d/cargo-edition-test

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -e
+top=`git rev-parse --show-toplevel`
+status=0
+for file in $(find "$top" -name Cargo.toml); do
+    if [ "$(toml get "$file" package.edition)" != '"2018"' ]; then
+        echo "$file is not using edition = "2018". Please ensure the field is\
+        present and includes the correct Rust edition."
```

Comment:
Sorry. I didn't mean wrap the source code. I mean to wrap the output. Output should always be less than the default size of a terminal (80 chars). Being concise can also help with that.

Reply:
I've split the output across two `echo` calls in my latest push, though the length of the filename could push it over in any case.